ProjectLighthouse/Database.cs

```diff
@@ -10,6 +10,7 @@
 using LBPUnion.ProjectLighthouse.Helpers;
 using LBPUnion.ProjectLighthouse.Levels;
 using LBPUnion.ProjectLighthouse.Levels.Categories;
+using LBPUnion.ProjectLighthouse.Logging;
```

Comment:
```suggestion
```
I think this might be a leftover from debug logging

Reply:
Resolved